### ISSUE TITLE
AzureOpenAI does not work

### ISSUE THREAD

**Issue by kocsszi** [truera/trulens#651]
This is the only solution to make Azure openAI work with the library inspite of it having support for it.
```python
from openai import AzureOpenAI as az
from trulens_eval.feedback.provider.openai import OpenAI
llm = OpenAI(model_engine="vanilla-gpt-35-turbo-16k")

oai_client  = az(
    api_key=OPEN_AI_KEY,  
    api_version="2023-10-01-preview",
    azure_endpoint = BASE_URL
    )

llm.endpoint.client = oai_client
```

using AzureOpenAI from trulens eval gives error as there is no deployment_id argument supported on openai side

**Comment by kocsszi** [comment]
duplicate of https://github.com/truera/trulens/pull/640

**Comment by kocsszi** [comment]
https://github.com/truera/trulens/pull/640

**Comment by joshreini1** [comment]
Hey @kocsszi ! Not exactly sure what happend with this open/closed issue, but we released a patch 0.18.3 that fixes support for AzureOpenAI. If you haven't already, please upgrade trulens_eval to 0.18.3.

Thanks!

**Comment by kocsszi** [comment]
Hey @joshreini1 ! Thanks for getting back, opened it then realized was duplicate and wanted to tag it if sy wanted a workaround. But the fix was pretty fast too.
